apps/src/p5lab/animationListModule.js

```diff
@@ -440,19 +457,25 @@ export function appendCustomFrames(props) {
  * Add a library animation to the project (at the end of the list, unless a spritelab project).
  * @param {!SerializedAnimation} props
  */
-export function addLibraryAnimation(props) {
+export function addLibraryAnimation(props, checkBackground) {
   return (dispatch, getState) => {
     const key = createUuid();
     if (getState().pageConstants && getState().pageConstants.isBlockly) {
       dispatch(addAnimationAction(key, props, 0));
     } else {
       dispatch(addAnimationAction(key, props));
     }
-    dispatch(
-      loadAnimationFromSource(key, () => {
-        dispatch(selectAnimation(key));
-      })
-    );
+    // if checkBackground, this means we don't want the selected animation to be a background
+    if (!checkBackground || !props.categories.includes('backgrounds')) {
```

Comment:
nit: maybe `skipBackground` instead of `checkBackground`?

now that we have this param, do we need to check `props.categories`? or can we just manipulate the param?

Reply:
we need to check props.categories to decide whether or not to set selected in the costume library. Otherwise, spritelab would never have default selected sprites showing up in the costume library (they'd have to scroll a bunch)